binaries/daemon/src/lib.rs

```diff
@@ -1567,7 +1567,7 @@ impl RunningDataflow {
         let running_nodes = self.running_nodes.clone();
         let grace_duration_kills = self.grace_duration_kills.clone();
         tokio::spawn(async move {
-            let duration = grace_duration.unwrap_or(Duration::from_millis(500));
+            let duration = grace_duration.unwrap_or(Duration::from_millis(2000));
```

Comment:
This change is not really related to this PR, is it? It's probably better to do it in a separate PR so users can see that the default was changed in the changelog.

Reply:
Sure.